adoc/MAIN.release-notes.adoc

```diff
@@ -109,6 +112,19 @@ SUSE update the ``user_data``. This should be removed as soon as possible after
 migration to {tf} 0.12.
 =========
 
+==== etcdctl
+Run `zypper in etcdctl` in the management host to install etcdctl.
+
+==== Update packages for general fixes
+Update skuba package and patterns-caasp-Management on your management workstation as you would do with any other package.
```

Comment:
Same here, please put in backtick quotes for monospace formatting of package names.